Source/Core/Core/HW/EXI/EXI_DeviceMemoryCard.cpp

```diff
@@ -488,7 +488,7 @@ void CEXIMemoryCard::DoState(PointerWrap& p)
 
 IEXIDevice* CEXIMemoryCard::FindDevice(TEXIDevices device_type, int customIndex)
```

Comment:
While you're at it, what about renaming `customIndex` to `custom_index` as well? You did so in the header, but not in the implementation